Assets/Python/RFCUtils.py

```diff
@@ -1573,25 +1572,26 @@ def canRespawn(self, iPlayer):
 			for tInterval in tResurrectionIntervals[iPlayer]:
 				iStart, iEnd = tInterval
 				if getTurnForYear(iStart) <= iGameTurn <= getTurnForYear(iEnd):
-					bPossible = True
 					break
+			else:

```

Comment:
That's really cool, I learned something today. And it's compatible with Python 2.4.